emission/net/ext_service/habitica/create_party_leaders_script.py

```diff
@@ -0,0 +1,38 @@
+import logging
+import emission.core.get_database as edb
+
+def create_party_leaders():
+    logging.basicConfig(level=logging.DEBUG)
+
+    ju_email = "ju.uemura@gmail.com"
+    ju_uuid = edb.get_uuid_db().find_one({'user_email': ju_email})['uuid']
+    logging.debug("Found Juliana's uuid %s" % ju_uuid)
+    edb.get_habitica_db().insert({'habitica_id': "e26ef1c3-d45e-464a-998c-729d107ecc42", 
```

Comment:
If you plan to call this multiple times as part of the tests, then you either need to delete these users as part of tearDown, or convert this to an upsert.